### ISSUE TITLE
Java init checks are a little unclear

### ISSUE THREAD

**Issue by machawk1** [N0taN3rd/wail#100]
### Are you submitting a **bug report** or a **feature request**?

Feature request

### What is the current behavior?

I am using a fairly fresh macOS 10.14 that does not have Java installed. For another tool, I installed Java 11. On WAIL's first-launch, it recognizes that Java 1.7 does not exist on the file system, asks if I want to install it to use Heritrix, then gives me an unclear warning when I state indicate "no".

### What is the expected behavior?
Ideally, Heritrix would use the installed version of Java (11 per above) but that is out of the scope of WAIL. Is Heritrix still limited to Java 7?

Expected behavior might be to indicate the severity of a user checking "no" to installing Java 7. The tone in the main window is that one can continue using WAIL without Heritrix where the popup explicitly states that WAIL cannot be used without Heritrix (which I believe to not be true, @N0taN3rd?).

The alert has an odd clarity issue, asking "Are you sure you don't want to install?" requires some pause to parse. Suggested revision for the alert window:

> The Heritrix archival crawler in WAIL requires JDK 1.7. WAIL requires Heritrix to perform crawls  (pending if this is still true).

> ...Run WAIL anyway? (NO) (YES)
> ...Use WAIL without installing Java? (NO) (YES)

### What's your environment?

macOS 10.14, WAIL v1.2.0-beta3, java 11.0.1 2018-10-16 LTS

### Other information

![screen shot 2018-10-22 at 10 10 32 am](https://user-images.githubusercontent.com/2514780/47297541-58dd7100-d5e3-11e8-9e45-fc43cb8e0b43.png)

**Comment by N0taN3rd** [comment]
Yeah that is kinda of a problem. WAIL electron needs a full rewrite.

**Comment by machawk1** [comment]
I asked a related question on the IIPC #heritrix Slack channel and @anjackson stated

> We run under 8.
> It should run under later versions but I’ve not tested anything else.

I am wondering if we should try to utilize the native Java if available and otherwise download the latest available for the platform. This would require testing Heritrix's capability on newer Java that nobody seems to have time for until then need Heritrix on their system. :D 

Rewrites take _a lot_ of cycles but in the interest of future endeavors, how would you do things differently if WAIL were rewritten?

I think, for this ticket, we can at least revise the verbiage.

**Comment by N0taN3rd** [comment]
Yeah I should have taged the other ticket about can not skip java install as well.
I think to use native Java would need to bump heritrix version.

The biggest part of the rewrite would be use prebuilt sql dists (npm packages) rather than the nedb, less windows etc.
Some TLC is needed:
- only start pywb, everything else as needed
- use prebuilt sql and move db stuff to main thread
- use workers with node-integration enabled (lots o work done electron side to make this happen nicer)
- Squidwarc is finally in a place where I do not have to use inception (web-view inside a browser) to do the browser based crawls
- node-warc can finally provide warc seed detection due to improvements in its parsing
- less windows
